src/screens/Sidebar/ReportBugs.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { View, WebView, ActivityIndicator } from 'react-native';
+import { connect } from 'react-redux';
+import { CannyBoard } from 'kitsu/components/CannyBoard';
+import { commonStyles } from 'kitsu/common/styles';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+class ReportBugs extends React.Component {
```

Comment:
I'm not sure we even need this wrapper component? Can this not all be stuffed into `CannyBoard` ?

Reply:
Are you suggesting that we fetch sso token in the component?

Reply:
yeah, is that not okay? i was thinking we just have one React.Component that contains all this code. if thats a bad idea we can leave this as is.